src/containers/SearchFilters/SearchFilters.js

```diff
@@ -55,32 +58,42 @@ class SearchFilters extends Component {
   ];
 
   renderMobileDrawer = (selectedCategory, setCategory, searchResults, counts, total) => {
-    const { classes, t } = this.props;
-    const { open }       = this.state;
+    const { classes, t }                    = this.props;
+    const { openMobileFilters, typeIsOpen } = this.state;
 
     return (
       <SwipeableDrawer
-        open={open}
-        onOpen={() => this.setState({ open: true })}
-        onClose={() => this.setState({ open: false })}
+        open={openMobileFilters}
+        onOpen={() => this.setState({ openMobileFilters: true })}
+        onClose={() => this.setState({ openMobileFilters: false })}
       >
         <div className={classes.drawer}>
           <div>
             <div className={classes.drawerHeader}>
               <Typography className={classes.header}>{t('filterBy')}</Typography>
```

Comment:
Voor translation keys gebruiken we snake_case

Reply:
goeie!

Reply:
Oh nee ik zie het al we hebben nu een mix van beide snake en camel maar het is beter om snake aan te houden. Moeten we later even aanpassen.